scripts/utils/bdlClassnameManager

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env node
+
+/* eslint-disable */
+const fs = require('fs');
+const colors = require('colors/safe');
+const { promisify } = require('util');
+const camelCase = require('lodash/camelCase');
+const kebabCase = require('lodash/kebabCase');
+const identity = require('lodash/identity');
+const invert = require('lodash/invert');
+const readFile = promisify(fs.readFile);
+const writeFile = promisify(fs.writeFile);
+
+/*
+    The BDL classname manager is a tool to help identify and correct the usage of classnames
+    names across box-ui-elements, and other projects.
+
+    USAGE
+
+    How to use this tool in "box-ui-elements"
+    (1) find . -name "*.scss" -exec ../scripts/utils/bdlClassnameManager swap {} \;
+    (2) find . -name "*.scss" -exec ../scripts/utils/bdlClassnameManager extend {} \;
+    (3) find . -not -name "*test.js" -and -name "*.js" -exec ../scripts/utils/bdlClassnameManager append {} \;
+
+    How to use this tool in other projects that have "box-ui-elements" as a dependency
+    (1) find . -name "*.scss" -exec ../scripts/utils/bdlClassnameManager swap {} \;
+    (2) find . -not -name "*test.js" -and -name "*.js" -exec ../scripts/utils/bdlClassnameManager swap {} \;
+
+    Please note that this tool will not catch all cases, so reviewing changes may be required.
+*/
+
+const conversionMap = {
+    // 'btn': 'bdl-Button',
+    // 'btn-group': 'bdl-ButtonGroup',
+    // 'btn-plain': 'bdl-Button--plain',
+    // 'btn-primary': 'bdl-Button--primary',
+    // 'is-disabled': 'bdl-is-disabled',
+    // 'label': 'bdl-Label',
+    'pill': 'bdl-Pill',
```

Comment:
I'm sticking to the core conflicting classnames to reduce the surface area of changes needed. I'm also starting with a partial list of classnames that are more rarely used in overrides in other codebases, to reduce risk for the first release.